README.md

```diff
@@ -35,8 +35,15 @@ go get github.com/go-spatial/go-wfs
 ## Running
 
 ```bash
+cd $GOPATH/src/github.com/go-spatial/go-wfs
+# create configuration
+cp go-wfs-config.toml local.toml
+vi local.toml  # update accordingly
+# server.url
+# collections.data
+
 # start server on http://localhost:9000/
-go run main.go  # or go build main.go
+go run main.go local.toml
```

Comment:
I'd like to keep the ability to run without having a config.  Perhaps add a port & data source flag with defaults for the current hard-coded values, plus an optional config flag to name the config file for those that want to tweak things.

Reply:
What's the downside of having a configuration?  Note that the configuration has important server metadata fields that will be used in the application. Leaving these default would create a unintended profile of applications with default configs.

The other benefit of having the config specified is for proper discovery when harvesting resources into catalogs.  The config forces users to provide copious metadata about their instance, which is a key component of spatial data infrastructures.

Reply:
Very significant benefit for developers/testers to be able to simply run the executable.  Good for admins that want to move quickly for short-lived or trial use to not have to worry about a lot of details.

I agree that resource discovery and metadata are good reasons to have a config option, but that we should't force it on people that are happy with what autoconfig can do.  I expect that autoconfig will evolve to be able to provide metadata from the resource.

One of the most frequent requests for tegola is a zero-config or auto-generated config to get it running quickly and easily.  I think we can reasonably provide a basic zero-config for easy entry, and have config functionality for people to step up to if appropriate for their needs.

Reply:
I think the best option here is to add a configuration that just works without any change for testers but can be customized for production usage

Reply:
One of the big sells Jeff makes on these packages is the ease of deployment.  Why add a required additional file when a couple of command-line flags suffice?